manifests/c/composer/composer/2.3.5/composer.composer.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-19041-1645
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: composer.composer
+PackageVersion: 2.3.5
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
```

Comment:
```suggestion
InstallerType: inno
```

Reply:
This is a bit confusing 🤔 
How should I choose between `exe` and `inno` next time?
I have not found a full list of possible values in the [docs](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.1.0/installer.md) either.